package.json

```diff
@@ -10,7 +10,9 @@
     "@types/node": "^16.11.36",
     "@types/react": "^18.0.9",
     "@types/react-dom": "^18.0.4",
+    "axios": "^0.27.2",
     "classnames": "^2.3.1",
+    "dotenv": "^16.0.1",
```

Comment:
I don't see it in use.

Reply:
thought, had to install it to use and create .env file